### DISCUSSION TITLE
Table

### DISCUSSION THREAD

**Discussion by donzee529** [requarks/wiki#3466] [Help / Questions]
**Question**
How can I add a table to my wiki?

**Comment by jeffrey-allan-brown** [comment]
It would depend on the type of editor you choose when creating the specific page you want the table on. I prefer to use markdown, in which case you could add a table like this:

| Header 1  | Header 2 | Header 3 | Header 4 | Header 5 |
|-----------|----------|----------|----------|----------|
| Something | Goes     | Here     |          |          |
|           |          |          | And      |          |
|           |          |          |          | Here     |

Otherwise you always have the option of choosing the HTML editor instead, and doing something like:

```html

<table width="100%">
<tr>
    <td width="25" class="em_aside">Something</td>
    <td width="25" class="em_aside">Something</td>
    <td width="25" class="em_aside">Something</td>
</tr>
</table>
```
